### PR TITLE
Updated Lift and Shift instructions for To Do List application

### PR DESCRIPTION
**Pull request by Abhijithang**
- Fixes for - VZ-3431, VZ-3830, VZ-3831.
- Removed ApplicationConfiguration and tls prefix from MySQL Yaml.
- Updated vz-application-modified.yaml to include MySQL Verrazzano components.
- Updated Lift and shift instructions to do the necessary changes.

### CODE REVIEW

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md [hunk 1]
```diff
@@ -315,7 +315,8 @@ the application into a Verrazzano environment.
 These include:
 1. Creating and labeling the `tododomain` namespace.
 1. Creating the necessary secrets required by the ToDo List application.
-1. Deploying MySQL to the `tododomain` namespace.
+1. Creating the Verrazzano components such as service, deployment, and configmap required by the MySQL instance in the `tododomain` namespace.
```
Comment (rosemarymarano) [inline content/en/docs/guides/lift-and-shift/lift-and-shift.md:318]: First sentence under Deploy to Verrazzano, please fix this typo:
...there are several steps required to deploy a the application -> ...there are several steps required to deploy the application
configmap -> ConfigMap

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md [hunk 2]
```diff
@@ -386,53 +387,94 @@ Update the generated `vz-application.yaml` file for the `todo` application to:
...
-#### Deploy MySQL
+#### Create Verrazzano components for MySQL  
 
-As noted previously, moving a production environment to Verrazzano would require migrating the
-data as well.  While data migration is beyond the scope of this guide, we will still need to
-include a MySQL instance to be deployed with the application in the Verrazzano environment.
+As noted previously, moving a production environment to Verrazzano would require migrating the data as well. While data migration is beyond the scope of this guide, we will still need to include a MySQL instance to be deployed with the application in the Verrazzano environment.
 
-To do so, download the [mysql-oam.yaml](../mysql-oam.yaml) file.
+ To do so, we need to first create the Verrazzano components by applying the [mysql-oam.yaml](../mysql-oam.yaml) in the `tododomain` namespace. The Verrazzano components will be deployed as Kubernetes objects when the To Do List application is deployed by applying the `vz-application.yaml` file is applied in the next step.
```
Comment (rosemarymarano) [inline content/en/docs/guides/lift-and-shift/lift-and-shift.md:444]: we need to first create -> first we need to create
[mysql-oam.yaml](../mysql-oam.yaml) -> [mysql-oam.yaml](../mysql-oam.yaml) file
by applying the `vz-application.yaml` file is applied in the next step. -> this sentence does not make sense and I am not sure how you want to correct it, but please do.
To Do List -> ToDo List

**Review by rosemarymarano — COMMENTED** [review]
Some edits.

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md (hunk 1 above)
Comment (Abhijithang) [reply content/en/docs/guides/lift-and-shift/lift-and-shift.md:318]: I think the typo "there are several steps required to deploy a the application" is in the production. Not part of the commit. I fixed it though. Thanks.

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md (hunk 2 above)
Comment (Abhijithang) [reply content/en/docs/guides/lift-and-shift/lift-and-shift.md:444]: I have corrected the text to : "To do so, first, we need to create the Verrazzano components for MySQL by applying the mysql-oam.yaml in the `tododomain` namespace. The components will be deployed as Kubernetes objects when the Todo List Application is deployed by applying the vz-application.yaml file in the next step. "

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/guides/lift-and-shift/lift-and-shift.md:318]: Thank you. I knew that but since you are editing the file I appreciate you fixing it.

Comment (Abhijithang) [reply content/en/docs/guides/lift-and-shift/lift-and-shift.md]: No problem, please let me know if there any issues with other parts of the document I will fix them as well. I will also review it.

**Review by markxnelson — APPROVED** [review]

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md [hunk 3]
```diff
@@ -441,7 +441,7 @@ $ diff vz-application.yaml vz-application-modified.yaml
 
 As noted previously, moving a production environment to Verrazzano would require migrating the data as well. While data migration is beyond the scope of this guide, we will still need to include a MySQL instance to be deployed with the application in the Verrazzano environment.
 
- To do so, we need to first create the Verrazzano components by applying the [mysql-oam.yaml](../mysql-oam.yaml) in the `tododomain` namespace. The Verrazzano components will be deployed as Kubernetes objects when the To Do List application is deployed by applying the `vz-application.yaml` file is applied in the next step. 
+To do so, first, we need to create the Verrazzano components for MySQL by applying the [mysql-oam.yaml](../mysql-oam.yaml) in the `tododomain` namespace. The components will be deployed as Kubernetes objects when the ToDo List application is deployed by applying the vz-application.yaml file in the next step.
```
Comment (rosemarymarano) [inline content/en/docs/guides/lift-and-shift/lift-and-shift.md:444]: the [mysql-oam.yaml](../mysql-oam.yaml)  -> the [mysql-oam.yaml](../mysql-oam.yaml) file

**Review by rosemarymarano — COMMENTED** [review]
Just one more minor fix, then I'll approve.

File: content/en/docs/guides/lift-and-shift/lift-and-shift.md (hunk 3 above)
Comment (Abhijithang) [reply content/en/docs/guides/lift-and-shift/lift-and-shift.md:444]: Sure, I will push this change.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**